modules/cpmso-yaml-failure-domain-azure.adoc

```diff
@@ -6,19 +6,28 @@
 [id="cpmso-yaml-failure-domain-azure_{context}"]
 = Sample Azure failure domain configuration
 
-The control plane machine set concept of a failure domain is analogous to existing Azure concept of an link:https://learn.microsoft.com/en-us/azure/azure-web-pubsub/concept-availability-zones[_Azure availability zone_]. The `ControlPlaneMachineSet` CR spreads control plane machines across multiple failure domains when possible. 
+The control plane machine set concept of a failure domain is analogous to existing Azure concept of an link:https://learn.microsoft.com/en-us/azure/azure-web-pubsub/concept-availability-zones[_Azure availability zone_]. The `ControlPlaneMachineSet` CR spreads control plane machines across multiple failure domains when possible.
 
 When configuring Azure failure domains in the control plane machine set, you must specify the availability zone name.
 
 .Sample Azure failure domain values
 [source,yaml]
 ----
 failureDomains:
-  azure: <1>
-  - zone: "1"
+  azure:
+  - subnet: <cluster_id>-master-subnet-0 <1>
+  - subnet: <cluster_id>-master-subnet-1
+  - subnet: <cluster_id>-master-subnet-2
+  - zone: "1" <2>
   - zone: "2"
   - zone: "3"
-  platform: Azure <2>
+  platform: Azure <3>
 ----
-<1> Each instance of `zone` specifies an Azure availability zone for a failure domain.
-<2> Specifies the cloud provider platform name. Do not change this value.
\ No newline at end of file
+<1> Each instance of `subnet` specifies a subnet that the installation program created. Each subnet corresponds to a zone. Do not change this value.
++
+[NOTE]
+====
+Multiple subnets support the NAT gateway outbound routing strategy, which is a Technology Preview feature in {product-title} {product-version}. Your cluster might not use this feature.
```

Comment:
This is techpreview in 4.14 because the CPMS work did not merge. It will likely be GA in 4.15 and default on new clusters.